build/cmake/modules/FindWebP.cmake

```diff
@@ -3,7 +3,7 @@ find_package(WebP CONFIG)
 if(NOT WebP_FOUND)
     find_package(PkgConfig QUIET)
     if(PKG_CONFIG_FOUND)
-        pkg_check_modules(WebP libwebp)
+        pkg_check_modules(WebP libwebpdemux)
```

Comment:
Sorry but don't we need both `-lwebp` and `-lwebpdemux`?

Reply:
`libwebpdemux` has an internal dependency on `libwebp` so it'll probably returns both libraries:
```
Requires.private: libwebp >= 0.2.0
```

But to be sure, you could add both of them to `pkg_check_modules`.

Reply:
This is a private dependency, so it's not taken into account when returning the libraries needed to be linked with, this can be seen by running pkg-config

```console
$ pkg-config --libs libwebpdemux
-lwebpdemux
```

After looking at this again, I'm pretty sure we need both `libwebpdemux` and `libwebp` here. @cjmayo Can you confirm that this works for you?